app/assets/javascripts/miq_application.js

```diff
@@ -320,7 +320,10 @@ function miqCheckForChanges() {
     }
   }
   // use default browser reaction for onclick
-  return true;
+  if(obj && (href = obj.getAttribute("href")))
```

Comment:
What I noticed was once a page was rendered by an Angular route/template, all the links on that page stopped working.
So that was the rationale behind this change.
This was one way to get the links to work again, but maybe there is a better way ... I'll look into it.